build/reactable.js

```diff
@@ -1075,6 +1069,8 @@ window.ReactDOM["default"] = window.ReactDOM;
                         }
 
                         switch (child.type) {
+                            case _thead.Thead:
+                                break;
```

Comment:
the build/ file gets out of sync all the time. Need to find a way to make Travis fail if that's the case.